src/cli.py

```diff
@@ -20,15 +20,16 @@
   -o=FILE,  --out=FILE              specify output image name
 
 Theme options:
-  --PALETTE[=LIST_COLOR_SET]        the palette can be found on the
+  --PALETTE[=LIST_COLOR_SET]        the palettes can be found in the
                                     src/palettes/ directory (actually there is
-                                    only nord), by replace the palette with the
-                                    name is possible to select the theme and if
+                                    only nord), by replacing 'PALETTE' with a
+                                    theme name it is possible to select a theme. If
```

Comment:
I think it can sound better, but I accept other suggestions here :smile: 

```suggestion
                                    only nord), by replacing 'PALETTE' with a
                                    theme name it is possible to select it. If
```

Reply:
I would agree that sounds better. I have made another commit with that change. Is it good to be merged now? Thanks for the feedback!